aggregations/db_tables/daily_new_accounts_by_app_count.py

```diff
@@ -0,0 +1,85 @@
+import datetime
+
+from . import DAY_LEN_SECONDS, daily_start_of_range, time_json
+from ..periodic_aggregations import PeriodicAggregations
+
+
+class DailyNewAccountsByAppCount(PeriodicAggregations):
```

Comment:
When you depend on some other table from near analytics database, you should define `dependencies` method like in the example I gave you: https://github.com/near/near-analytics/blob/main/aggregations/db_tables/daily_new_unique_contracts_count.py#L9-L10